docs/tutorials/multimodal/matching/index.rst

```diff
@@ -3,41 +3,41 @@ Matching
 .. container:: cards
 
    .. card::
-      :title: Text-to-text Matching with AutoMM - Quick Start
+      :title: Text-to-text Semantic Matching with AutoMM
       :link: text2text_matching.html
 
-      How to use AutoMM for text to text matching.
+      How to use AutoMM for text-to-text semantic matching.
    
    .. card::
-      :title: Image-to-image Matching with AutoMM - Quick Start
+      :title: Image-to-image Semantic Matching with AutoMM
```

Comment:
You can do another pass to ensure that the title in the cards are the same as the title of the individual notebooks.

Reply:
Checked.